### PR TITLE
Added Ecobee and Nest to features.

### PR DESCRIPTION
**Pull request by watou**
Adding to 2.0 compatible action and bindings as in #3614.

### CODE REVIEW

File: features/openhab-addons-external/pom.xml [hunk 1]
```diff
@@ -38,6 +38,7 @@
                                 <artifact><file>src/main/resources/conf/comfoair.cfg</file><type>cfg</type><classifier>comfoair</classifier></artifact>
                                 <artifact><file>src/main/resources/conf/denon.cfg</file><type>cfg</type><classifier>denon</classifier></artifact>
                                 <artifact><file>src/main/resources/conf/dmx.cfg</file><type>cfg</type><classifier>dmx</classifier></artifact>
+                                <artifact><file>src/main/resources/conf/ecobee.cfg</file><type>cfg</type><classifier>dmx</classifier></artifact>
```
Comment (kaikreuzer) [inline features/openhab-addons-external/pom.xml]: classifier must be ecobee, not dmx!

File: features/openhab-addons-external/pom.xml [hunk 2]
```diff
@@ -55,6 +56,7 @@
                                 <artifact><file>src/main/resources/conf/milight.cfg</file><type>cfg</type><classifier>milight</classifier></artifact>
                                 <artifact><file>src/main/resources/conf/modbus.cfg</file><type>cfg</type><classifier>modbus</classifier></artifact>
                                 <artifact><file>src/main/resources/conf/mysql.cfg</file><type>cfg</type><classifier>mysql</classifier></artifact>
+                                <artifact><file>src/main/resources/conf/nest.cfg</file><type>cfg</type><classifier>dmx</classifier></artifact>
```
Comment (kaikreuzer) [inline features/openhab-addons-external/pom.xml]: dito

**Comment by kaikreuzer** [comment]
Cool, I see you are following closely the latest changes without having to tell you anything about it :-)

**Comment by watou** [comment]
Trying to get up to speed on the latest, anyway!  Fixed those 2 oversights.

**Comment by kaikreuzer** [comment]
Thanks! FYI: I hope to be able to announce this way of adding new 1.x addons to OH2 next Monday.

**Comment by watou** [comment]
I'm looking forward to Beta 1 and all of the improvements!

**Comment by kaikreuzer** [comment]
Also cherry-picked for 1.8.

**Comment by watou** [comment]
It looks like  I broke [the build](https://openhab-ci.innoq.io/jenkins/view/openHAB/job/openhab-1-rc/) with this PR, but I'm not sure how or how to fix it.  Sorry! :sweat: 

> [WARNING] Feature resolution failed for [openhab-action-ecobee/1.8.0.SNAPSHOT]
Message: Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=openhab-action-ecobee; type=karaf.feature; version=1.8.0.SNAPSHOT; filter:="(&(osgi.identity=openhab-action-ecobee)(type=karaf.feature)(version>=1.8.0.SNAPSHOT))" [caused by: Unable to resolve openhab-action-ecobee/1.8.0.SNAPSHOT: missing requirement [openhab-action-ecobee/1.8.0.SNAPSHOT] osgi.identity; osgi.identity=org.openhab.action.ecobee; type=osgi.bundle; version="[1.8.0.201601070234,1.8.0.201601070234]"; resolution:=mandatory [caused by: Unable to resolve org.openhab.action.ecobee/1.8.0.201601070234: missing requirement [org.openhab.action.ecobee/1.8.0.201601070234] osgi.wiring.package; filter:="(osgi.wiring.package=org.openhab.binding.ecobee)"]]

**Comment by watou** [comment]
Possibly due to missing dependency to `<feature>openhab-binding-ecobee</feature>`.  Building now.

**Pull request merged**